k8s/apps/redis.yml

```
@@ -0,0 +1,42 @@
+kind: Service
+metadata:
+  name: mindjunkies-redis-master
+  labels:
+    app: mindjunkies-redis
+spec:
+  ports:
+    - port: 6379
+      targetPort: 6379
+  selector:
+    app: mindjunkies-redis
+  type: ClusterIP
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)